components/brave_wallet_ui/components/desktop/views/explore_web3/web3_dapp_details.tsx

```diff
@@ -4,20 +4,16 @@
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
 import * as React from 'react'
-import { Redirect, useParams } from 'react-router'
-import ProgressRing from '@brave/leo/react/progressRing'
+import { DialogProps } from '@brave/leo/react/dialog'
 
 // Hooks
-import {
-  useGetMainnetsQuery,
-  useGetTopDappsQuery
-} from '../../../../common/slices/api.slice'
+import { useGetMainnetsQuery } from '../../../../common/slices/api.slice'
 
 // Utils
 import Amount from '../../../../utils/amount'
 import { CurrencySymbols } from '../../../../utils/currency-symbols'
 import { getLocale } from '../../../../../common/locale'
-import { WalletRoutes } from '../../../../constants/types'
+import { BraveWallet } from '../../../../constants/types'
```

Comment:
can you use an type import for this? Also move to a `// types` or `// constants` import group
```suggestion
import type { BraveWallet } from '../../../../constants/types'
```